lib/async/http/protocol/http1/response.rb

```diff
@@ -30,6 +31,10 @@ def initialize(connection, version, status, reason, headers, body)
 					def connection
 						@connection
 					end
+
+					def reason
```

Comment:
Can you use `attr`?

Also, can you add a test (a basic unit test is sufficient).

Reply:
Changed to use attr_reader and added a couple test cases.